workspaces/answer-corrector/index.test.ts

```diff
@@ -1,3 +1,4 @@
+import 'jest';
```

Comment:
hmm, is it necessary to import jest? I'm not saying it's incorrect, I've just never seen this before

Reply:
I have imported it to remove this error:
`Cannot find name 'describe'`.
But let's try another solution ;)